corehq/apps/app_manager/models.py

```diff
@@ -1983,6 +2005,19 @@ def wrap(cls, data):
 
         return super(AdvancedForm, cls).wrap(data)
 
+    def _pre_delete_hook(self):
+        try:
+            self.disable_schedule()
+        except (ScheduleError, TypeError, AttributeError):
```

Comment:
no logging / notification? when would these exceptions be thrown / why doesn't it matter if they are?